src/main/webapp/login.jsp

```diff
@@ -13,6 +13,9 @@
       <a href="${loginUrl}">
         <button>Log In</button>
         </a>
+      <a href="create-new-user.jsp">
```

Comment:
you'd want to send them to the servlet instead of to the jsp, since the servlet has the logic for authentication checks